### PR TITLE
python packages: updated minor version libraries

### PR DESCRIPTION
**Pull request by domdinicola**

### CODE REVIEW

File: src/requirements/input/base.in [hunk 1]
```diff
@@ -10,37 +10,37 @@ djangorestframework-jwt==1.11.0
...
-django-debug-toolbar==1.9.1
+django-cors-headers==2.4
+django-debug-toolbar==1.9.1  # update requires Django upgrade
 django-easy-pdf==0.1.1
 django-filter==1.0.2
 django-fsm==2.6
-django-import-export==1.0
-django-leaflet==0.23
-django-logentry-admin==1.0.3
+django-import-export==1.1
```
Comment (robertavram) [inline src/requirements/input/base.in]: are we still using this one?

File: src/requirements/input/base.in [hunk 2]
```diff
@@ -1,6 +1,6 @@
 azure-storage==0.20.2
-carto==1.2.2
-celery==4.2
+carto==1.3.1
```
Comment (robertavram) [inline src/requirements/input/base.in]: Did we do a hands-on test for these updates:
tenant-schemas-celery
and
carto

File: src/requirements/input/base.in (hunk 1 above)
Comment (domdinicola) [reply src/requirements/input/base.in]: Unit model (Reports app) is using it in the admin.

Comment (domdinicola) [reply src/requirements/input/base.in]: Partner admin, Partner export, Intervention and so on

Comment (robertavram) [reply src/requirements/input/base.in]: All admin exports can be deprecated.. I don't think a lot of them work. Good to know the tests are good.

Thanks Dom! 👍

**Pull request merged**